platforms/azure/giantnetes/main.tf

```diff
@@ -106,6 +106,7 @@ locals {
     "LogentriesToken"          = "${var.logentries_token}"
     "MasterCount"              = "${var.master_count}"
     "OIDCEnabled"              = "${var.oidc_enabled}"
+    "OIDCIssuerURL"            = "https://${var.oidc_issuer_dns}.${var.base_domain}"
```

Comment:
Can't we hardcode `dex.g8s` here? What's the point of extracting `var.oidc_issuer_dns`?

Reply:
Don't like to have it here in case we change issuer